src/SourceBuild/content/eng/bootstrap/buildBootstrapPreviouslySB.csproj

```diff
@@ -52,43 +54,43 @@
           Returns="@(PackageDownload)">
     <ItemGroup>
       <!-- Generate a cross-product between runtime packs and Linux RIDs -->
```

Comment:
There are several references to `Linux RID[s]` in this file. Consider updating those to `Unix`.